rotkehlchen/tests/utils/accounting.py

```diff
@@ -241,6 +243,61 @@ def upload_csv_and_check(
         offset += 1
 
 
+def _assert_empty_computed_value(
+        cost_basis: 'CostBasisInfo',
+        computed_value: Optional[list],
+) -> None:
+    """Checks an zero/empty values rendered in google sheets."""
+    if computed_value is None:
+        return  # None is fine for an empty value
+
+    if cost_basis.is_complete is False:
+        assert computed_value == ['Incomplete cost basis information for spend.']
+    else:
+        assert computed_value == []
+
+
+def upload_csv_and_check_cost_basis(
```

Comment:
Why are you duplicating the logic of `upload_csv_and_check`? If you want to add checks for another column(cost basis) just add them there.